bin/MadGraph5_aMCatNLO/cards/production/13TeV/tt0123j_1l_t_5f_ckm_LO_MLM/tt0123j_1l_t_5f_ckm_LO_MLM_madspin_card.dat

```diff
@@ -0,0 +1,33 @@
+#************************************************************
+#*                        MadSpin                           *
+#*                                                          *
+#*    P. Artoisenet, R. Frederix, R. Rietkerk, O. Mattelaer * 
+#*                                                          *
+#*    Part of the MadGraph5_aMC@NLO Framework:              *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+#
+#directory for gridpack mode
+set ms_dir ./madspingrid
+
+#initialization parameters
+ set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
+# set BW_cut 15                # cut on how far the particle can be off-shell
+ set max_weight_ps_point 400  # number of PS to estimate the maximum for each event
+# 
+#to properly limit the number of concurrent processes for grid running
+set max_running_process 1
+#
+
+define ell+ = e+ mu+ ta+
+define ell- = e- mu- ta-
+
+# specify the decay for the final state particles
+decay t > w+ b, w+ > ell+ vl
+decay t~ > w- b~, w- > j j
```

Comment:
not 100% sure j will include the b's in this case.

Might want to explicitly define the jets (or define a "q" multiparticle here) to be sure.